src/Integration.Vsix.UnitTests/Analysis/IssueHandlerTests.cs

```diff
@@ -79,34 +79,30 @@ public void HandleNewIssues_UpdatedSnapshotHasExpectedValues()
         }
 
```

Comment:
Here I also mostly inverted tests or in some cases even took out issuefilter tests as they were testing if the list has changed which, now it no longer does. I added 1 test to check if the issues get marked correctly as suppressed correctly